lib/dscli/source.rb

```diff
@@ -1,6 +1,6 @@
 class Source < Thor
 
-  desc 'list [page]', 'Lists all managed sources'
+  desc 'list [page]', 'Lists all Managed Sources'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.